simulation-runner/README-GENERATING-AGENT-SIMULATIONS.md

```diff
@@ -282,15 +282,15 @@ puppet-gatling-jenkins-plugin project.
 12. In order for Gatling to generate useful reports per request endpoint, the
     names of the endpoints should be renamed.
 
-  | Name                 | Endpoint     |
-  | -------------------- | ------------ |
-  | catalog              | /production/catalog/agent.localdomain
-  | filemeta pluginfacts | /production/file_metadatas/pluginfacts
-  | filemeta plugins     | /production/file_metadatas/plugins
-  | filemeta             | /production/file_metadatas/modules/xyz
-  | filemeta mco plugins | /production/file_metadatas/modules/pe_mcollective/plugins
-  | node                 | /production/node/agent.localdomain
-  | report               | /production/report/agent.localdomain
+  | Name                 | Legacy Endpoint     | Modern v3 Endpoint |
+  | -------------------- | ------------ | ------------ |
```

Comment:
Nitpick, but I think it would be worth lining up the separators between the second and third columns for improved readability.